inc/dbconnection.class.php

```diff
@@ -54,6 +54,32 @@ static function getTypeName($nb=0) {
 
 
    /**
+    * Create GLPI main configuration file
+    *
+    * @since 9.1
```

Comment:
Yes : bad version, but actually, it has been released for GLPi 9.1 first :) 

I'll change this if the PR is considered acceptable.